server/server.go

```diff
@@ -260,7 +260,7 @@ func (svr *Server) HandlePut(w http.ResponseWriter, r *http.Request) (commitment
 		}
 	}
 
-	svr.log.Info(fmt.Sprintf("write commitment: %x\n", comm))
```

Comment:
oh yeah I noticed this as well. Not super sure what the intent of this was. If this is only meant for keccak commitments, then we should print it in the if clause above probably? Or was this just a mistake @epociask ?

Reply:
I think there is a bit confusion on the term commitment schema, https://github.com/Layr-Labs/eigenda-proxy?tab=readme-ov-file#commitment-schemas. The readme section actually means returned commitment-schemas. 

In the commented log above, it only prints comm used when you send data to eigenda-proxy. If it is optimism alt da, the commitment is left empty, so nothing gets printed. 

I often find it useful to debug, if I have the response commitment, which is something rollup will commit onchain. So I am adding this change

Reply:
The responseCommitment should be same as comm when keccak is used right? So there's literally no point in logging `comm`? If that's the case, then let's just merge this. But if there is a difference between `comm` and `responseCommit`, then perhaps we should log both?

Reply:
no really, for keccak, you append a version byte 0 before the hash, https://github.com/Layr-Labs/eigenda-proxy/blob/main/commitments/op.go#L110

Reply:
since it is a byte, printing will just show 00 in the front

Reply:
oh comm is the commit type?? I thought it was the commitment for keccak commitments, where the commitment is supplied in the POST request?